.github/workflows/helm.yaml

```diff
@@ -13,13 +13,28 @@ jobs:
         name: Package Helm chart
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
             - uses: actions/setup-python@v2
               with:
                   python-version: '>=3.9.0 <4.0.0'
             - run: python3 -m pip install -r ci/cilib/requirements.txt
 
             - run: helm lint charts/substra-frontend
+
+            - uses: azure/setup-helm@v3.3
```

Comment:
`helm lint` uses lint, but maybe from the distro and not the desired version?

Reply:
Moved the install before the lint. It's better to all use the same version, right ?

Reply:
Yes 👍